README.md

```diff
@@ -1,3 +1,25 @@
-# InterviewFlashCards
+# Interview Flash Cards
+An open-source project in the form of iOS technical, data structures, and algorithm questions in flashcard form.
```

Comment:
`An open-source project designed to be a tool for iOS developers preparing for an interview. The project consists of data structure and algorithm questions, as well as iOS technical questions, all in flashcard form.`